modules/user/server-ts/getEmailTemplate.ts

```diff
@@ -0,0 +1,40 @@
+import { UserShape } from './sql';
+import settings from '../../../settings';
+
+interface EmailData {
+  user?: UserShape;
+  url: string;
+}
+
+const { app } = settings;
+
+const getEmailTemplate = (type: string, { user, url }: EmailData) => {
+  switch (type) {
```

Comment:
Try to avoid using a switch. Make it more modular instead.

Reply:
Fixed